### ISSUE TITLE
NVDA fails to report system input language switching with Alt+Shift

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#8419]
### Steps to reproduce:
1. Update NVDA to the latest "next" snapshot (next-15324,255cb03f).
2. Try to switch between input languages with Alt+Shift. My input languages are 2: English U.S. and Bulgarian Bulgaria.

### Expected behavior:
NVDA should report the input language change.

### Actual behavior:
NVDA only plays its error sound.

### System configuration:
- Operating system: Windows 10 Pro version 1803 (build 17134.112), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-15324,255cb03f.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

### Other questions:
I'm attaching the NVDA log with the logging level set to "Debug".
[NVDA_Log_2018-06-19.log](https://github.com/nvaccess/nvda/files/2114101/NVDA_Log_2018-06-19.log)

**Comment by leonardder** [comment]
@dkager, could you please provide a fix for this ASAP?

**Comment by dkager** [comment]
The problem appears to be that the language contains non-ASCII characters. This did not show up in testing because I only used Western European layouts.

@k-kolev1985 Could you try [this build](https://ci.appveyor.com/api/buildjobs/dx7uiegr3k88idvp/artifacts/output%2Fnvda_snapshot_pr8401-15333%2C834b1a86.exe) to confirm if it fixes the problem?

**Comment by k-kolev1985** [comment]
@dkager Yes, this test build fixes the issue. Now NVDA correctly reports the names of the input languages when I switch to each of them.

And I've internally guessed correctly that the problem has something to do with text encoding, taking in consideration the error about Unicode encoding. Problems like this sometimes happen when non-latin (cyrillic) text is involved. 😀

**Comment by leonardder** [comment]
This was fixed by @dkager in #8401 and will be available in the next snapshot that's currently being build.
